app/models/manageiq/providers/redhat/infra_manager/refresh_parser.rb

```diff
@@ -658,4 +659,10 @@ def self.datacenter_inv_to_hashes(inv, cluster_uids, vm_uids, storage_uids, host
 
     result
   end
+
+  private_class_method
+
+  def self.vm_memory_reserve(vm_inv)
+    vm_inv[:memory_policy] && vm_inv[:memory_policy][:guaranteed]
```

Comment:
vm_inv.class is Ovirt::Vm
so I will have to use 
vm_inv.attributes.fetch_path(:memory_policy, :guaranteed)
and I couldn't decide which is better, what do you think?